worker/worker_blackbox_test.go

```diff
@@ -18,23 +22,41 @@ import (
 
 type WorkerTestSuite struct {
 	gormtestsupport.DBTestSuite
+	application application.Application
+	ctx         context.Context
 }
 
 func TestWorker(t *testing.T) {
 	suite.Run(t, &WorkerTestSuite{DBTestSuite: gormtestsupport.NewDBTestSuite()})
 }
 
+// SetupSuite overrides the DBTestSuite's function but calls it before doing anything else
+// The SetupSuite method will run before the tests in the suite are run.
+// It sets up a database connection for all the tests in this suite without polluting global space.
+func (s *WorkerTestSuite) SetupSuite() {
+	s.DBTestSuite.SetupSuite()
+	s.ctx = migration.NewMigrationContext(context.Background())
+	s.application = gormapplication.NewGormDB(s.DB, s.Configuration, nil)
+}
```

Comment:
This override is not needed either.

Reply:
fixed in c3c911f

Reply:
I don't need to define that method at all since it just calls s.DBTestSuite.SetupSuite()

Reply:
fixed in 1ee9935